packages/typespec-ts/test/modularUnit/scenarios/example/example.md

```diff
@@ -93,8 +94,17 @@ export function _readSend(
   id: string,
   options: ReadOptionalParams = { requestOptions: {} },
 ): StreamableMethod {
+  const path = expandUrlTemplate(
+    "/{id}",
+    {
+      id: id,
+    },
+    {
+      allowReserved: options?.requestOptions?.skipUrlEncoding,
```

Comment:
is there a case where some parameters need to allowReserved but others don't ?

Reply:
parameter-level allowReserved is only supported in url template and operation-level allowReserved is https://github.com/Azure/autorest.typescript/blob/a165ea8e19ea7e97148655787456bbc9be7b4053/packages/typespec-ts/test/modularUnit/static/urlTemplate.spec.ts#L345.

Reply:
issue tracked here: https://github.com/Azure/azure-sdk-for-js/issues/33301